test/tasks/mock/example/task-05/config.toml

```diff
@@ -0,0 +1,7 @@
+# this is the file used to determine the network configuration
```

Comment:
Can we change the location of this file to the new layout and give it a descriptive name? i.e. `ExampleDevnetTransferOwnerTemplate.toml`